zendriver/cdp/network.py

```diff
@@ -1491,6 +1491,9 @@ def to_json(self) -> T_JSON_DICT:
 
     @classmethod
     def from_json(cls, json: T_JSON_DICT) -> CookiePartitionKey:
+        if isinstance(json, str):
+            # Some chrome versions return partitionKey as string
+            return cls(json, False)
```

Comment:
just out of curiosity are there any docs on this?

Reply:
I didn't find something in docs, simple logged CDP message and got from practice usage of chrome >=129